tools/scripts/generateSoundMetadata.rb

```diff
@@ -20,7 +20,9 @@
 begin
   require 'taglib'
 rescue LoadError
-  puts "You need taglib installed locally to run this script. Run 'brew install taglib' then 'gem install taglib-ruby'."
+  puts "You need taglib installed locally to run this script. Run 'brew install taglib' then 'gem install taglib-ruby'.\n"\
+"If that doesn't work, add \"gem 'taglib-ruby'\" to your Gemfile and run 'bundle install' "\
+"(do not commit changes to Gemfile or Gemfile.lock)"
```

Comment:
You could optionally use the Heredoc syntax for big multiline strings like this, instead of assembling it manually:

```suggestion
  puts <<~HEREDOC
    You need taglib installed locally to run this script. Run 'brew install taglib' then 'gem install taglib-ruby'.
    If that doesn't work, add "gem 'taglib-ruby'" to your Gemfile and run 'bundle install'
    (do not commit changes to Gemfile or Gemfile.lock)
  HEREDOC
```

Reply:
Oh cool thanks for the tip! Will add.